packages/manager/src/features/Linodes/LinodeCreate/Tabs/StackScripts/StackScriptSelectionRow.tsx

```diff
@@ -60,18 +61,19 @@ export const StackScriptSelectionRow = (props: Props) => {
           </Stack>
         </label>
       </TableCell>
-      <TableCell
-        sx={{
-          display: 'flex',
-          justifyContent: 'flex-end',
-          minWidth: 120,
-          paddingRight: 0,
-          height: 44,
-        }}
-        actionCell
-      >
-        <InlineMenuAction actionText="Show Details" onClick={onOpenDetails} />
+      <TableCell noWrap sx={{ paddingRight: 0 }}>
+        <StyledRootContainer>
+          <InlineMenuAction actionText="Show Details" onClick={onOpenDetails} />
+        </StyledRootContainer>
```

Comment:
I think it would be ideal if we could use `actionCell`  because it's a global pattern and avoid a styled container if possible.

Looks like this accomplishes the same behavior:

```suggestion
      <TableCell actionCell>
        <InlineMenuAction actionText="Show Details" onClick={onOpenDetails} />
```

Reply:
Thanks @bnussman-akamai, with the initial reference of [InlineMenuAction](https://github.com/linode/manager/blob/develop/packages/manager/src/features/ObjectStorage/BucketLanding/BucketActionMenu.tsx#L36) i was wrapping it with `StyledRootContainer` but yes with the suggestion of your of simply passing the actionCell will also work [reference](https://github.com/linode/manager/blob/develop/packages/manager/src/features/ObjectStorage/BucketDetail/ObjectTableRow.tsx#L68)